liquibase-standard/src/main/java/liquibase/resource/DirectoryPathHandler.java

```diff
@@ -33,6 +38,13 @@ public ResourceAccessor getResourceAccessor(String root) throws FileNotFoundExce
         root = root
                 .replace("file:", "")
                 .replace("\\", "/");
+        try {
+            // Because this method is passed a URL from liquibase.resource.ClassLoaderResourceAccessor.configureAdditionalResourceAccessors,
+            // it should be decoded from its URL-encoded form.
+            root = decode(root, StandardCharsets.UTF_8.name());
+        } catch (UnsupportedEncodingException e) {
+            Scope.getCurrentScope().getLog(getClass()).fine("Failed to decode path " + root + "; continuing without decoding.", e);
```

Comment:
But if it is a `file://`  url should we not suppress this error message as we did the replace above?

Reply:
Line 44 is primarily about removing things like `%20` from the `root` and replacing them with a space. I think keeping the error message could be useful.